projects/sunbird-video-player/src/lib/sunbird-video-player.component.ts

```diff
@@ -255,6 +253,12 @@ export class SunbirdVideoPlayerComponent implements OnInit, AfterViewInit, OnDes
       this.isFullScreen = false;
     }
     this.showQumlPlayer = true;
+    this.viewerService.raiseImpressionEvent('interactive-question-set');
```

Comment:
How can we get the details of to which question-set, navigation happened?
Do we have QuestionSet details in the Cdata?

@HarishGangula

Reply:
Share the same events

Reply:
@vinukumar-vs 
We are sending two events, one is Impression when quml-player opens
`{
  "edata": {
    "type": "workflow",
    "subtype": "",
    "pageid": "interactive-question-set",
  }
}`


and other is the Interact event when we are interacting with the video marker(interception point)
`{
  "type": "TOUCH",
  "subtype": "",
  "id": "video_marker_selected",
  "pageid": "videostage",
  "extra": {
    "questionSetId": "do_213272808198291456121",
    "type": "interactive-video",
    "interceptedAt": 50
  }
}`
here I've added type as `interactive-video` to identify the question-set is played within interactive video content.
cc: @HarishGangula